ROADMAP.md

```diff
@@ -524,6 +524,7 @@ Since the list is too long, now we just put those already supported options here
 | :white_check_mark: :star: | :vsp {file}       | Split vertically current window in two.                                 |                                                                                                                        |
 | :white_check_mark:        | &lt;ctrl-w&gt; s  | Split current window in two.                                            |                                                                                                                        |
 | :white_check_mark: :star: | &lt;ctrl-w&gt; v  | Split vertically current window in two.                                 |                                                                                                                        |
+| :white_check_mark: :star: | &lt;ctrl-w&gt; o  | Close other editor groups                                               |                                                                                                                        |
```

Comment:
Was about to merge, then realized that the other descriptions end in a period. Hate to be that guy, but could you add one?

Reply:
Do you squash on merge or do you need it on the same commit?

Reply:
Squash on merge, don't worry about it. Thanks 🙂